assets/css/refills/_tooltip.scss

```diff
@@ -0,0 +1,65 @@
+.tooltip-item {
+  $base-border-color: gainsboro !default;
+  $base-border-radius: 3px !default;
+  $base-line-height: 1.5em !default;
+  $dark-gray: #333 !default;
+  $base-font-color: $dark-gray !default;
+  $tooltip-background: white;
+  $tooltip-color: $base-font-color;
+  $tooltip-max-width: 32em;
+  $tooltip-arrow-width: 8px;
+  $tooltip-shadow: 0 2px 2px silver;
+  $tooltip-distance-from-item: 3em;
+  $tooltip-arrow-distance-from-box: -1.3em;
+
+  // background: $tooltip-background;
+  // border: 1px solid $base-border-color;
+  // border-radius: $base-border-radius;
+  display: inline-block;
```

Comment:
Properties should be ordered position, display, padding, text-align

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)